recipe/setup.patch

```diff
@@ -1,17 +1,21 @@
-diff --git setup.py setup.py
-index 4ee0ac3..6617197 100755
---- setup.py
-+++ setup.py
-@@ -145,12 +145,8 @@ setup_params = dict(
+--- setup.py	2016-08-02 10:40:52.000000000 -0300
++++ setup.py	2016-08-02 16:41:40.950478555 -0300
+@@ -165,18 +165,6 @@
+         Topic :: System :: Systems Administration
          Topic :: Utilities
          """).strip().splitlines(),
-     extras_require={
+-    extras_require={
 -        "ssl:sys_platform=='win32'": "wincertstore==0.2",
 -        "certs": "certifi==2016.2.28",
-     },
-     dependency_links=[
--        'https://pypi.python.org/packages/source/c/certifi/certifi-2016.2.28.tar.gz#md5=5d672aa766e1f773c75cfeccd02d3650',
--        'https://pypi.python.org/packages/source/w/wincertstore/wincertstore-0.2.zip#md5=ae728f2f007185648d0c7a8679b361e2',
-     ],
+-    },
+-    dependency_links=[
+-        pypi_link(
+-            'certifi-2016.2.28.tar.gz#md5=5d672aa766e1f773c75cfeccd02d3650',
+-        ),
+-        pypi_link(
+-            'wincertstore-0.2.zip#md5=ae728f2f007185648d0c7a8679b361e2',
+-        ),
+-    ],
      scripts=[],
      tests_require=[
+         'setuptools[ssl]',
```

Comment:
Should probably re-name the patch to avoid showing false positives. This is not an update in the patch, but differences in the code base.